### PR TITLE
connecting the dots

### PR DESCRIPTION
**Pull request by mailtime-wing**

### CODE REVIEW

File: src/components/ConnectingTheDots/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+              values={{
+                span: str => (
+                  <span css={theme => [subTitleStyle(theme), subTitle2(theme)]}>
+                    {str}
+                  </span>
+                ),
+              }}
+            />
+          </span>
+        </p>
```
Comment (chochihim) [inline src/components/ConnectingTheDots/index.js]: This snippet is quite duplicated. Please change to something like
```
<p css={theme => [subTitleStyle(theme), subTitle1(theme)]}>
  <FormattedMessage
    id="providesActionableConsumer"
    defaultMessage={`A <span>Data Dashboard</span> that provides actionable consumer insights based on transactional data.`}
    values={{
      span: str => (
        <span css={subTitle2}>
          {str}
        </span>
      ),
    }}
  />
</p>
```

File: src/components/ConnectingTheDots/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        <p css={theme => [titleStyle(theme), title1(theme)]}>
+          <FormattedMessage
+            id="connectingTheDots"
+            defaultMessage="Connecting the dots with"
+          />
+        </p>
+        <p css={theme => [titleStyle(theme), title2(theme)]}>
+          <FormattedMessage
+            id="alternativeData"
+            defaultMessage="ALTERNATIVE DATA"
```
[the remark this answers is not in the document]
Comment (mailtime-wing) [reply src/components/ConnectingTheDots/index.js]: combined.

File: src/components/ConnectingTheDots/index.js (hunk 1 above)
Comment (mailtime-wing) [reply src/components/ConnectingTheDots/index.js]: fixed.

File: src/components/ConnectingTheDots/index.js [hunk 3]
```diff
@@ -16,40 +16,34 @@ import {
...
-        </p>
-        <p css={theme => [titleStyle(theme), title2(theme)]}>
-          <FormattedMessage
-            id="alternativeData"
-            defaultMessage="ALTERNATIVE DATA"
+            id="connectingTheDotsWithAlternativeData"
+            defaultMessage="Connecting the dots with <p>ALTERNATIVE DATA</p>"
+            values={{
+              p: str => (
+                <p css={theme => [titleStyle(theme), title2(theme)]}>{str}</p>
```
Comment (chochihim) [inline src/components/ConnectingTheDots/index.js]: `p` cannot be under `p`

**Pull request merged**